internal/ui/tabs.go

```diff
@@ -31,7 +32,11 @@ func Create(app fyne.App, window fyne.Window) *container.AppTabs {
 			tabs.SelectIndex(0)
 		}
 
-		send.newTransfer(uris)
+		confirm := dialog.NewConfirm("Create using custom code", "Create using custom code?", func(custom bool) {
+			send.client.CustomCode = custom
+			send.newTransfer(uris)
+		}, window)
+		confirm.Show()
 	})
```

Comment:
This can be simplified (and we can avoid some duplicated wording):
```suggestion
		dialog.ShowConfirm("Custom Code", "Use a custom code?", func(custom bool) {
			send.client.CustomCode = custom
			send.newTransfer(uris)
		}, window)
	})
```

Reply:
Updated